orbit/cmd/orbit/shell.go

```diff
@@ -98,6 +105,19 @@ var shellCommand = &cli.Command{
 			osquery.WithFlags([]string{"--database_path", osqueryDB}),
 		}
 
+		// If fleetd is built with the --fleet-certificate flag then orbit already starts osqueryd with
```

Comment:
Sorry, I meant `orbit.go` already starts osqueryd with --tls_server_certs.

So what's missing here is that if `c.String("fleet-certificate") != ""` then we should use that here.